docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,162 @@
+-rw-rw-r--. 1 mkotelni mkotelni 154 Apr 29 16:30 README.md
+```
+
+- Add the policy to the relevant table in the [README.md](./open-policy-agent/README.md) file.
+
+```
+### Networking
+Policy  | Description | Prerequisites
+------- | ----------- | -------------
+[httpsonly](./networking/httpsonly.yaml) | Ensures that there are *no* http routes |
+```
+
+### Kyverno
+
+- In order to create a Kyverno policy, create a new directory under the relevant compliance control in the [kyverno directory](../kyverno). Make sure to create a readable name for the policy directory.
+
+```
+$ mkdir ./kyverno/networking/httpsonly
+```
+
+- Create a `<policy-name>.yaml` file. The file will contain the policy definition. Make sure to create a readable name for the policy.
+
+```
+$ ls -l ./kyverno/networking/httpsonly/
+-rw-rw-r--. 1 mkotelni mkotelni 199 Apr 29 16:30 httpsonly.yaml
+```
+
+- Create a `README.md` file that briefly explains the policy, and how to use it. Instructions on how to create a `README.md` file for policies can be found [here](./).
+
+```
+$ ls -l ./kyverno/networking/httpsonly/
+-rw-rw-r--. 1 mkotelni mkotelni 199 Apr 29 16:30 httpsonly.yaml
+-rw-rw-r--. 1 mkotelni mkotelni 154 Apr 29 16:30 README.md
+```
+
+- Add the policy to the relevant table in the [README.md](./kyverno/README.md) file.
+
+```
+### Networking
+Policy  | Description | Prerequisites
+------- | ----------- | -------------
+[httpsonly](./networking/httpsonly.yaml) | Ensures that there are *no* http routes |
+```
+
+### Red Hat Advanced Cluster Management for Kubernetes
+
+- In order to create a Red Hat Advanced Cluster Management policy, create a new directory under the relevant compliance control in the [redhat-acm directory](../redhat-acm). Make sure to create a readable name for the policy directory.
+
+```
+$ mkdir ./redhat-acm/networking/gatekeeper-allow-httpsonly
+```
+
+- Create a `<policy-name>.yaml` file. The file will contain the policy definition. Make sure to create a readable name for the policy.
+
+```
+$ ls -l ./redhat-acm/networking/gatekeeper-allow-httpsonly/
+-rw-rw-r--. 1 mkotelni mkotelni 199 Apr 29 16:30 gatekeeper-allow-httpsonly.yaml
+```
+
+- Create a `README.md` file that briefly explains the policy, and how to use it. Instructions on how to create a `README.md` file for policies can be found [here](./).
+
+```
+$ ls -l ./redhat-acm/networking/gatekeeper-allow-httpsonly/
+-rw-rw-r--. 1 mkotelni mkotelni 199 Apr 29 16:30 httpsonly.yaml
+-rw-rw-r--. 1 mkotelni mkotelni 154 Apr 29 16:30 README.md
+```
+
+- Add the policy to the relevant table in the [README.md](./redhat-acm/README.md) file.
+
+```
+### Networking
+Policy  | Description | Prerequisites
+------- | ----------- | -------------
+[gatekeeper-allow-httpsonly](./networking/gatekeeper-allow-httpsonly.yaml) | Ensures that there are *no* http routes | The [GateKeeper](https://github.com/open-policy-agent/gatekeeper) operator needs to be installed
+```
+
+### Contributing
```

Comment:
@michaelkotelnikov 
In this case the PR is from forked master to upstream master. 
I suggest to create PR from the feature/branch of the forked repo to the upstream master so this will enable contributors to create and work on multiple policies - I will mention this in PR rules.